pkgs/by-name/ar/artifacts-credprovider/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchurl,
+  stdenvNoCC,
+}:
+stdenvNoCC.mkDerivation rec {
+  name = "artifacts-credprovider";
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/microsoft/artifacts-credprovider/releases/download/v${version}/Microsoft.Net8.NuGet.CredentialProvider.tar.gz";
+    hash = "sha256-WEssfdOKMjJ3WD/egD4wA69k+JdB9O/ZWM8RstRJGkA=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r netcore $out/bin
```

Comment:
Missing `preInstall` and `postInstall`. It doesn't seem to be an executable, are you sure you are putting it to the correct place?

Reply:
Sorry, rushed this last night before bed. Going to clean up.

Reply:
Refactored to use buildDotnetPackage which looks like it outputs it to `lib/dotnet/${pname}` which is more in line with it's plugin usage.